ui/helpers/location.js

```diff
@@ -25,6 +25,18 @@ export const setLocation = function setLocation(params, hashPrefix = '/jobs') {
   window.location.hash = `#${hashPrefix}${createQueryParams(params)}`;
 };
 
+// change the url hash without firing a ``hashchange`` event.
+export const replaceLocation = function replaceLocation(
+  params,
+  hashPrefix = '/jobs',
+) {
+  window.history.replaceState(
```

Comment:
I'm curious about the use of window.history instead of using react-routers location object that also contains history and I think some sort of replace or update state?